src/vs/workbench/contrib/notebook/browser/view/cellParts/foldedCellHint.ts

```diff
@@ -67,6 +73,40 @@ export class FoldedCellHint extends CellContentPart {
 		return expandIcon;
 	}
 
+	private getRunFoldedSectionButton(range: ICellRange): HTMLElement {
+		const runAllContainer = DOM.$('span.folded-cell-run-section-button');
+		const cells = this._notebookEditor.getCellsInRange(range);
+
+		const isRunning = cells.some(cell => {
+			const cellExecution = this._notebookExecutionStateService.getCellExecution(cell.uri);
+			return cellExecution && cellExecution.state === NotebookCellExecutionState.Executing;
+		});
+
+		const runAllIcon = isRunning ?
```

Comment:
maybe we can have a toolbar/menu for it, so we can support run, stop and spinner.

Reply:
Tracked in https://github.com/microsoft/vscode/issues/206465